ankihub/gui/utils.py

```diff
@@ -774,3 +775,37 @@ def get_ah_did_of_deck_or_ancestor_deck(anki_did: DeckId) -> Optional[uuid.UUID]
 
 def logged_into_ankiweb() -> bool:
     return bool(aqt.mw.pm.sync_auth())
+
+
+def refresh_anki_ui() -> None:
+    changes = OpChanges(
+        notetype=True,
+        note_text=True,
+        browser_table=True,
+        browser_sidebar=True,
+    )
+    # Anki's UI componennts subscribe to this hook to refresh themselves when needed.
+    # This is the mechanism that Anki uses to update the UI when its backend makes changes to the collection.
+    gui_hooks.operation_did_execute(
+        changes=changes,
+        handler=None,
+    )
+
+    # Study queue related UI is refreshed when focus is not None.
+    # This prevents the deck browser from fading out when the main window temporarily loses focus.
```

Comment:
I remember seeing the fade-out effect getting stuck sometimes but I can't reproduce it now.

Does calling `mw.fade_in_webview()` instead of constructing an OpChanges here work? That's more intuitive and less reliant on Anki's internal handling.

Reply:
> I remember seeing the fade-out effect getting stuck sometimes but I can't reproduce it now.#

You mean on this PR branch?

Reply:
I mean generally in Anki when progress dialogs are shown. Didn't manage to reproduce it in this branch yet.